templates/problem/list.html

```diff
@@ -287,6 +290,15 @@ <h3>{{ _('Hot problems') }} <i class="fa fa-fire"></i></h3>
                         {% if not request.in_contest %}
                             <td class="ac-rate">{{ problem.ac_rate|floatformat(1) }}%</td>
                         {% endif %}
+                        {% if problem.has_public_editorial and not request.in_contest %}
```

Comment:
I think we should hide this column outright in contest (similar to AC rate).

Reply:
Yeah that was my intent, I'm just bad at logic.